spec/unit/services/users/omniauth_spec.rb

```diff
@@ -1,12 +1,25 @@
 require 'rails_helper'
 
 describe Users::Omniauth do
-  let(:auth) { mock_github_provider }
+  context 'when signing with Github' do
+    let(:auth) { mock_oauth_provider('github') }
 
-  describe 'user creation' do
-    it 'returns a user' do
-      user = Users::Omniauth.new(auth).get
-      expect(user).to be_a_kind_of(User)
+    context 'when user created' do
+      it 'returns a user' do
+        user = Users::Omniauth.new(auth).get
+        expect(user).to be_a_kind_of(User)
+      end
+    end
+  end
+
+  context 'when signing with Github' do
```

Comment:
@sinpai should be `'when signing with Google'`

Reply:
👍